### PR TITLE
[P4-1435] Updates `PersonSerializer` to use `Person` attributes

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

P4-1435

### What?

We've updated a `Person` to include a bunch of attributes that were
previously on the profile.

This change makes it so that these attributes are now available to
clients of the api via the `PersonSerializer`

- [x] Consume [new](https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/493) attributes in `PersonSerializer`
- [x] Fix broken factory to reflect move to using `Person` for these attributes

### Why?

Beyond cleanup, this is the final step in having our apis use `Person` attributes instead of `Profile` attributes in the `PeopleController`

This change **does not affect what the frontend receives** and is transparent.

### CODE REVIEW

**Comment by willfish** [comment]
We cannot merge this until the following PRs have been merged:

https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/512
https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/499

Followed by running the following rake task on every environment:

https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/496

**Comment by willfish** [comment]
I'll run the integration tests against this branch once we've merged in https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/512

File: spec/serializers/person_serializer_spec.rb [hunk 1]
```diff
@@ -94,28 +95,25 @@
     end
 
     before do
-      profile = person.latest_profile
-      profile.profile_identifiers = profile_identifiers
-      profile.save!
+      person.update(criminal_records_office: nil, police_national_computer: 'ABC123456', prison_number: 'XYZ123456')
```
Comment (cwrw) [inline spec/serializers/person_serializer_spec.rb:98]: maybe we can define a new `let(:person)` with those values for this context instead of an update?

**Comment by willfish** [comment]
Superseded by https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/565

**Pull request closed without merging**